backend/app/organisation/routers.go

```diff
@@ -11,4 +11,10 @@ func OrganisationRouter() {
 	router.POST("/organisations", OrganisationsAllHandler)
 	router.GET("/organisations/{organisation_id}", OrganisationsHandler)
 	router.PATCH("/organisations/{organisation_id}", OrganisationsHandler)
+
+	router.GET("/articles", GetOrganisationArticlesHandler)
+	router.POST("/articles", CreateArticleHandler)
+	router.GET("/articles/:articleID", GetArticleHandler)
+	router.PUT("/articles/:articleID", UpdateArticleHandler)
+	router.DELETE("/articles/:articleID", DeleteArticleHandler)
```

Comment:
Because articles are linked to an organisation, i was thinking about scoping that...
So instead having:
```
/articles/{article_id}....
```
we should have : 
```
/organisations/{organisation_id}/articles/{article_id}....
```
for all endpoints.

Reply:
@gtindo what do you think about this API design ?